dgraphtest/load.go

```diff
@@ -80,17 +80,28 @@ func readGzData(r io.Reader, encryption bool) ([]byte, error) {
 		}
 	}
 
-	gr, err := gzip.NewReader(r)
-	if err != nil {
+	var rr io.Reader
+	if gr, err := gzip.NewReader(r); err != nil &&
+		strings.Contains(err.Error(), "gzip: invalid header") {
+		if len(sf) > 0 {
+			if rr, err = os.Open(sf); err != nil {
```

Comment:
I don't think we need to do this. And we do decide to keep it, we should close the file in defer.